### ISSUE TITLE
Submitting devel version to CRAN

### ISSUE THREAD

**Issue by jimhester** [trinker/pacman#113]
Are there any plans to submit the devel version of pacman to CRAN?

We will be sending a new version of remotes and devtools to CRAN which will cause a failure in the current CRAN release of pacman.

However the devel version of pacman which depends on remotes works with only minor modification https://github.com/trinker/pacman/pull/112

**Comment by trinker** [comment]
@jimhester We took a look and merged.  My co-auther @Dasonk is going to do some additional testing this weekend.  Our goal is to push to CRAN monday evening or Tuesday morning.  Will that work for you?

**Comment by jimhester** [comment]
Yes that is fine, thanks! Also we submitted remotes 2.0.0 yesterday and it is on CRAN now.
